.git-blame-ignore-revs

```diff
@@ -1 +1,10 @@
+# Following revs are all whitespace changes; use with
+# git blame --ignore-revs-file .ignore-revs <...>
```

Comment:
shouldn't it be `git blame --ignore-revs-file .git-blame-ignore-revs`?

Reply:
fixed